surf/apps/communities/serializers.py

```diff
@@ -61,12 +54,16 @@ def get_materials_count(obj):
     def get_publish_status(obj):
         return str(PublishStatus.get(obj.publish_status))
 
+    @staticmethod
+    def get_details(obj):
+        details = CommunityDetail.objects.filter(community=obj).all()
+        return [CommunityDetailSerializer(detail).data for detail in details]
```

Comment:
It's more canonical to use nested serializers: https://www.django-rest-framework.org/api-guide/relations/#writable-nested-serializers

Reply:
Yeah I've been doing this way too complicated